driver/src/main/java/oracle/nosql/driver/iam/SecurityTokenSupplier.java

```diff
@@ -15,6 +15,7 @@
 import java.io.IOException;
 import java.net.URI;
 import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
```

Comment:
With this change Charset is no longer used, correct?

Reply:
Yes